pre_install_report/README.md

```diff
@@ -146,6 +146,10 @@ web-viewable, HTML format.
 You can modify the <tool-download-dir>/viya4-ark/pre_install_report/viya_deployment_settings.ini file to alter the   
 minimum and aggregate settings for CPU and memory on nodes. For more information, see the details in the file.
 
+If you modify the VIYA_K8S_VERSION_MIN to a version less than the minimum Kubernetes version supported by this 
+release of the report tool, you are operatiing outside the supported capabilities of the report tool.  It is better 
```

Comment:
"It is better to use a release of Viya 4 ARK tools matching the..." should be SAS recommends using a release of Viya 4 ARK tools that matches the..."

Reply:
Done.  Thank you @FredPerrySAS .